host.json

```diff
@@ -1,16 +1,11 @@
 {
-    "version": "2.0",
     "functionTimeout": "00:10:00",
-    "extensions": {
-        "eventHubs": {
-            "batchCheckpointFrequency": 1,
-            "eventProcessorOptions": {
-                "maxBatchSize": 64,
-                "prefetchCount": 128
-            }
-        }
+    "eventHub": {
+      "maxBatchSize": 64,
+      "prefetchCount": 256,
```

Comment:
prefetchCount increased to 256 - is that on purpose? (if just reverting to v1, should it be kept at 128)

if this is for optimization, we're making a lot of changes around that - this may make it harder to see which changes helped and which didn't, no?